new-packages/ts-project/src/index.ts

```diff
@@ -162,7 +147,28 @@ export function createProject(
         return [];
       }
       return findCreateMachineCalls(ts, sourceFile).map((call) => {
-        return extractMachineConfig(ts, call, sourceFile);
+        const ctx: ExtractionContext = {
+          sourceFile,
+          version,
```

Comment:
Can we call this more specifically like `xstateVersion`?

Reply:
sure